include/swift/AST/DiagnosticsSIL.def

```diff
@@ -352,6 +352,10 @@ ERROR(global_must_be_compile_time_const,none,
       "global variable must be a compile-time constant", ())
 ERROR(non_final_generic_class_function,none,
       "classes cannot have non-final generic fuctions in embedded Swift", ())
+ERROR(embedded_swift_existential_type,none,
+      "using values of protocol type %0 is not allowed in embedded Swift", (Type))
+ERROR(embedded_swift_existential,none,
+      "using values of protocol types is not allowed in embedded Swift", ())
```

Comment:
How about `cannot use a value of protocol type %0 in embedded Swift`? this aligns better with the rest of the diagnostic we have.

Reply:
That sounds great, let's go with that. Thanks!